geomstats/learning/mdm.py

```diff
@@ -0,0 +1,99 @@
+"""The MDM classifier on manifolds."""
+
+import geomstats.backend as gs
+from geomstats.learning.frechet_mean import FrechetMean
+
+
+class RiemannianMinimumDistanceToMeanClassifier():
+    """
+    Classifier implementing the MDM scheme on manifolds.
+
+    Attributes
+    ----------
+    riemannian_metric : string or callable.
+        The distance metric to use.
+    n_clusters: int, number of clusters.
+    mean_estimate: array-like, shape=[n_classes, n_features] if
+    point_type='vector'
+                   shape=[n_classes, n_features, n_features]
+                    if point_type='matrix'
+       Frechet means of each class.
+    """
+
+    def __init__(
+            self,
+            riemannian_metric,
+            n_clusters,
+            point_type='matrix'):
+        self.riemannian_metric = riemannian_metric
+        self.n_clusters = n_clusters
+        self.point_type = point_type
+        self.mean_estimate = None
```

Comment:
Should be `self.mean_estimate_` to be consistent with scikit-learn conventions:

Estimated Attributes
Attributes that have been estimated from the data must always have a name ending with trailing underscore, for example the coefficients of some regression estimator would be stored in a coef_ attribute after fit has been called.

The estimated attributes are expected to be overridden when you call fit a second time.

From: https://scikit-learn.org/stable/developers/develop.html

Reply:
Done.